src/main/webapp/item_pages/donation-form.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="DonationForm.css">
```

Comment:
If the stylesheet is empty, it doesn't really need to exist yet. It's good practice to avoid having too many unused files and code in the production codebase (the `main` branch in this case). You can keep it if you think it'll make future development easier, but make sure the file name uses the same naming scheme as the other css files

Reply:
I changed the naming scheme and I had created it earlier on for future use. It now has code in it.